druid-shell/src/platform/windows/window.rs

```diff
@@ -121,6 +125,14 @@ pub struct WindowHandle {
     state: Weak<WindowState>,
 }
 
+enum BlockingOp {
```

Comment:
I would prefer the name "DeferredOp" but won't block on it (see several other comments).